recipes/vigra/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set version = "1.11.0" %}
+
+package:
+  name: vigra
+  version: {{ version }}
+
+source:
+  {% set version_tag = "Version-%s" % version.replace(".", "-") %}
+  fn: vigra-{{ version_tag }}-src.tar.gz
+  url: https://github.com/ukoethe/vigra/archive/{{ version_tag }}.tar.gz
+  sha1: 67788d181e759e625769bf0678887718d3dc02e2
+
+build:
+  skip: true  # [win]
```

Comment:
Windows support is possible. However, it has been disabled. The reason being VIGRA requires C++11 support or at the bare minimum some of the C++0x features. As MSVC compilers supported for CPython by and large do not support C++11, this makes it impossible to build VIGRA for older versions of CPython due to AFAICT and arbitrary constraint. In any event, until we can figure out a way forward it is simply easier to disable Windows support for now.